components/content-service/BUILD.yaml

```diff
@@ -28,6 +28,7 @@ packages:
       - GOOS=linux
     config:
       packaging: app
+      buildCommand: ["go", "build", "-trimpath", "-ldflags='-buildid= -w -s'"]
```

Comment:
You can do this centrally for all Go packages in https://github.com/gitpod-io/gitpod/blob/main/WORKSPACE.yaml#L10

This way only `workspacekit` would need some adjustment.

Reply:
fixed